src/examples/wasm/server.go

```diff
@@ -3,15 +3,26 @@ package main
 import (
 	"log"
 	"net/http"
+	"os"
+	"path/filepath"
 	"strings"
 )
 
-const dir = "./html"
+var dir = "main"
```

Comment:
This confuses me. If we serve the main folder, we also have to change the compilation instructions everywhere. Why is this done?

Reply:
Ahhh.  And now I notice the Makefile.  Doh. :man_facepalming:

Reply:
We should probably adjust the Makefile to not create the `html` directory in the `clean` target, as that means people need to run `make clean` before either `make main` or `make export` will work.

Maybe the right approach is to have the `html` directory be kept around (eg add a blank `.keep` file in there), and use something like `rm -f html/*` to clear it out instead?

Reply:
And nope.  The clean target gets run first anyway.

This is turning out to be one of those weeks. :frowning: